src/zeep/client.py

```diff
@@ -77,14 +77,16 @@ def __getitem__(self, key):
 class Client(object):
 
     def __init__(self, wsdl, wsse=None, transport=None,
-                 service_name=None, port_name=None, plugins=None):
+                 service_name=None, port_name=None, plugins=[]):
```

Comment:
This is actually pretty bad practice, see http://docs.python-guide.org/en/latest/writing/gotchas/#mutable-default-arguments